src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+// class Storage - handles the loading of data when duke is run and
+// saving of data when duke is closed
+class Storage {
+    private String filePath;
+
+    public Storage(String s) {
+        this.filePath = s;
+    }
+
+    public ArrayList<Task> load() {
+        System.out.println("    . . . Loading . . . ");
```

Comment:
You should try to abstract out the `println` statements to your `Ui` class if you have implemented it.